src/Users/UserProfile.js

```diff
@@ -1,19 +1,25 @@
 import React from "react";
 import { connect } from 'react-redux';
+import UserOrdersCompleted from './UserOrdersCompleted';
 
-const UserProfile = () => {
+const UserProfile = ({auth}) => {
     return (
-        <hr />
-    )
-};
-const mapState = state => {
-    return {
 
-    }
+    <div>
+    <h1>
+    {auth.username}
+    </h1>
+     <p>{auth.imageUrl}</p>
```

Comment:
Img tag if it doesn't work

Reply:
https://github.com/users/bob-zs/projects/1/views/1